backend/src/service/userService.js

```diff
@@ -0,0 +1,16 @@
+const firebaseService = require("./firebaseService")
+const geoService = require("./geoService")
+
+module.exports = {
+    //check if the user is logged
+    async verifyUserToken(idToken) {
+        const uid = await firebaseService.auth(idToken)
+        return uid;
```

Comment:
why not using a direct return here?

Reply:
It is the same thing that happens with the storeLength variable, I changed in the last commit